embedly/tests.py

```diff
@@ -0,0 +1,143 @@
+import json
+import unittest
+import urllib
+
+import mock
+
+import embedly
+
+
+class TestEmbedlyProxy(unittest.TestCase):
+
+    def _get_url_data(self, url):
+        return {
+            'original_url': url,
+        }
+
+    def _get_urls_data(self, urls):
+        return [self._get_url_data(url) for url in urls]
+
+    def _build_query_url(self, urls):
```

Comment:
D'oh, this is building the URL to request our proxy api, the one in embedly.py is for requesting the embedly API, two different call signatures.  Disregard this.